packages/hooks/src/useWalletsList.ts

```diff
@@ -94,7 +94,7 @@ const useWalletsList = () => {
                 is_dark_mode_on ? '--dark' : ''
             }`;
             const wallet_icon = currency_to_icon_mapper[wallet_currency_type];
-
+            const fiat_currencies = ['USD', 'EUR', 'AUD', 'GBP'];
```

Comment:
Should we exclude `GBP`? Because it's not supposed to be there in the first place.